### PR TITLE
Chore: Typo fixed in multiple files of docs/source/en/model_doc

### PR DESCRIPTION
**Pull request by SusheelThapa**
# What does this PR do?

This PR fix the typo present in the files inside `docs/source/en/model_doc` folder

```pseudo
optimizaton > optimization
Similiar > Similar
exaclty > exactly
conditionning > conditioning
outupt > output
avalable > avaliable
```

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by a-r-r-o-w — APPROVED** [review]

File: docs/source/en/model_doc/nllb-moe.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ which means that tokens have less probability of being forwarded. Moreover, if a
 states (kind of like a residual connection) while they are masked in `NLLB`'s top-2 routing mechanism. 
 
 ## Generating with NLLB-MoE
-The avalable checkpoints requires around 350GB of storage. Make sure to use `accelerate` if you do not have enough RAM on your machine.
+The avaliable checkpoints requires around 350GB of storage. Make sure to use `accelerate` if you do not have enough RAM on your machine.
```
Comment (a-r-r-o-w) [inline docs/source/en/model_doc/nllb-moe.md:56]:
```suggestion
The available checkpoints require around 350GB of storage. Make sure to use `accelerate` if you do not have enough RAM on your machine.
```

**Comment by SusheelThapa** [comment]
@a-r-r-o-w  Can you review again?

**Comment by ArthurZucker** [comment]
Hey! Feel free to ping me when this is ready!

**Comment by SusheelThapa** [comment]
@ArthurZucker Can you review this PR and provide me necessary feedback?

**Review by ArthurZucker — APPROVED** [review]
Good typo corrections thanks!

**Pull request merged**